core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -484,6 +487,14 @@
         TRANSPORT_TYPE = type;
     }
 
+    private static final Path DEFAULT_MULTIPART_LOCATION;
+
+    static {
+        DEFAULT_MULTIPART_LOCATION = Paths.get(get("multipartLocation",
+                                                   System.getProperty("java.io.tmpdir"),
```

Comment:
How about setting a subfolder for multipart?
```suggestion
                                                   System.getProperty("java.io.tmpdir") + "/armeria/multipart-files",
```